.github/ISSUE_TEMPLATE/backport_request.yml

```diff
@@ -0,0 +1,111 @@
+name: "Backport to Stable"
+description: "Create a backport request for a package that is up-to-date in the unstable channel, but outdated in the stable channel."
+title: "Backport to Stable: PACKAGENAME OLDVERSION → NEWVERSION"
+labels: ["0.kind: enhancement", "9.needs: port to stable"]
+body:
+  - type: "markdown"
+    attributes:
+      value: |
+        <p align="center">
+          <a href="https://nixos.org">
+            <picture>
+              <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/NixOS/nixos-homepage/main/public/logo/nixos-hires.png">
+              <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/NixOS/nixos-artwork/master/logo/nixos-white.png">
+              <img src="https://raw.githubusercontent.com/NixOS/nixos-homepage/main/public/logo/nixos-hires.png" width="400px" alt="NixOS logo">
+            </picture>
+          </a>
+        </p>
+
+        > [!CAUTION]
+        > **Before you begin:** Be advised that backports are subject to the [release suitability guidelines](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#changes-acceptable-for-releases). As a rule of thumb, Nixpkgs Stable will almost never recieve breaking changes in the form of major package updates, but minor updates and bugfixes should be backported when possible.
+        > Please read the rules above carefully before filing this backport request.
+
+        Welcome to Nixpkgs. Please replace the **`Backport to Stable: PACKAGENAME OLDVERSION → NEWVERSION`** template above with the correct package name (As seen in the [NixOS Package Search](https://search.nixos.org/packages)), the current version of the package in Nixpkgs Stable and the current version of the package in Nixpkgs Unstable.
```

Comment:
I've always felt like the whole UPPERCASE means variable thing had contributed to user confusion. Maybe we should switch to `<angle-brackets>` instead? (Feedback/debate welcome)

Reply:
> I think this can be quite confusing as <nixpkgs> can be easily misread as <package>...

Is the main reason why I kept the UPPERCASE variables.